build/room_ensurer/room_ensurer.py

```diff
@@ -303,7 +303,7 @@ def _ensure_admin_power_levels(self, room_id: Optional[str], room_alias: str) ->
             return
 
         log.info(f"Ensuring power levels for {room_alias}")
-        api = self._apis[self._own_server_name]
+        api = self._own_api
```

Comment:
Do we need to change the definition of `self._own_api` to use `get` as well?

Reply:
No, there's a check in the line before the assignment.